### ISSUE TITLE
Fuzzy search in app suite + translation

### ISSUE THREAD

**Issue by realgooseman** [Goodwy/Dialer#57]
Hi @Goodwy,

Don't know if you're a solo developer or if it's a team. Regardless, amazing app suite!

### **Fuzzy search**

One annoying thing I noticed with the dialer app is that the search is not very effective. It needs a precise search or has to start with exact country code to match a contact. Implementing fuzzy search in all of your calling/message related apps would improve overall experience.

This could be done by allowing to search for typed number chunk in full contact numbers for faster results and also ignore country codes because some numbers are saved without them.

For example in France the international country code is +33. People there have contacts saved in international country code format but some are also saved in national format (meaning the +33 international country code is replaced by a 0). Both work for calling the contact but depending if you saved contacts with international country code format or national format, they might or might not appear on the current Right Dialer's search results.

### **Translation panel**

Setting up a translation panel like **[Crowdin](https://crowdin.com/)** or **[Weblate](https://weblate.org/)** would allow the community to add and improve translation strings for the app. The submission is free for open source projects on both platforms, I find Crowdin to be easier to use.

**Comment by Goodwy** [comment]
We're searching on the part you specified. If you don't know which code the contact is recorded with, just search without the country code.

**Comment by Goodwy** [comment]
As an option you can ignore the first character zero when searching, but I'm not sure that it will work in all countries.

**Comment by realgooseman** [comment]
Yes, it shows results when typing number without local prefix "0" for me. It's a temporary fix but not convenient for long term. Search logic should be improved in the future.

**Comment by mmoole** [comment]
Continuing the example from above: when typing `735` (without local prefix, that would be my use case since I have memorized that part of the number most, and it's easier to type than a part of the name. When trying this out, it doesn't display the contact on my device. But it would be more helpful if numbers matching partly like this would also be displayed 👍
